internal/service/cloudbackupsnapshotexportbucket/resource_cloud_backup_snapshot_export_bucket_test.go

```diff
@@ -69,6 +73,65 @@ func basicTestCase(tb testing.TB) *resource.TestCase {
 	}
 }
 
+func basicAzureTestCase(t *testing.T) *resource.TestCase {
+	t.Helper()
+
+	var (
+		projectID          = acc.ProjectIDExecution(t)
+		tenantID           = os.Getenv("AZURE_TENANT_ID")
+		bucketName         = os.Getenv("AZURE_BLOB_STORAGE_CONTAINER_NAME")
+		serviceURL         = os.Getenv("AZURE_SERVICE_URL")
+		atlasAzureAppID    = os.Getenv("AZURE_ATLAS_APP_ID")
+		servicePrincipalID = os.Getenv("AZURE_SERVICE_PRINCIPAL_ID")
+	)
+
+	return &resource.TestCase{
+		PreCheck: func() {
+			acc.PreCheckBasic(t)
+			acc.PreCheckCloudProviderAccessAzure(t)
+			acc.PreCheckAzureExportBucket(t)
+		},
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		CheckDestroy:             checkDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: configAzureBasic(projectID, atlasAzureAppID, servicePrincipalID, tenantID, bucketName, serviceURL),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					checkExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "project_id", projectID),
```

Comment:
Can we use `acc.AddAttrChecks` to remove duplication?

Reply:
Done